src/main/java/ca/on/oicr/gsi/somnus/Server.java

```diff
@@ -240,6 +241,18 @@ protected void renderContent(XMLStreamWriter writer) throws XMLStreamException {
                   os.write(mapper.writeValueAsBytes(result));
                 }
               }
+
+              break;
+            case "GET":
+              ArrayNode inhibitionDump = mapper.createArrayNode();
+              for (Inhibition i : inhibitions) {
```

Comment:
Would it make sense to not export expired inhibitions here? That would prevent manually expired inhibitions being reloaded in an active state.

Reply:
Yes! Forgot expired inhibitions are still in memory. I'll fix this

Reply:
Fixed